api-reference/v1.0/resources/groups_overview.md

```diff
@@ -0,0 +1,273 @@
+# Working with groups in Microsoft Graph
+
+Groups are collections of [users](*) and [devices](*) (and more?) who share access to resources in Office365 or in your app. All operations in Microsoft Graph on groups require administrator consent. Using Microsoft Graph, you can perform the following operations and more:

```

Comment:
Agreed - and this is part of the challenge because we have many group types.  I would open with the traditional view of groups and then extend.  Something like:  "Traditionally, groups are logical containers or collections of users, that are most commonly used to control access to resources - whether this is to Microsoft services or to your own app.  Microsoft Graph provides you with APIs to create and manage a variety of different types of groups and group functionality to suit your scenario needs."
Somehow we need to continue this to really draw the reader to use Office Groups, and when it's appropriate to use the other group types.  We really need to make this definitive guidance IMHO